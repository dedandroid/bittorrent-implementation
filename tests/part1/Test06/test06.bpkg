ident:86292a8906ea57d8f62c8cd149ffde1554f7e70edf8265a6bd239f4d94d62eb72ae02fefe724cca89db3c0054db6c7ee1cf3bd3945e1189b54e24e78c3f8ad6b95f532e77c88513a61ca043c7241fac80bd3dca483dd408a245288efa209cd1c8ff6baa3d71189cad1c69a53c6c83d5bc417ecbb3cdb5952a183be674207f52c94380e33a0ff94236a618c9ce93ee6a7adfac3d63c6d1807278b3172ab081a0b7f5322a6f030837aaf5e1d0b8049a4413956bfcaffa7211d17b24bdcb15559692bfdab89a20c4295947df49a6efb7649136cee5816558ea131e257db5c7c2b54fb0da56bbb039a4cb2f09cce83abeffdbfb54100c035a825a154d1254d03cf07ebc3cc38c6e6e0b911de2f37c4a93a026d5298b6e9cd986a938b2b2ffc83785d5aff2a5031dc93626ee9a089d0c4811eb0ddb2ee3bbdf1f50deae63b60ff10dd0aabc8af45c36c869707d42441351022add65759c2c3e597caf9e1d230740a6a740cb66882b674d3ecdceae1b65b0c67063bc934cea3387144e2ec3929525b951d0e64222d655d7915cf1f948e6d902ad3837b698cfd966ab29c220a07897a44dc747def9dc233ce56a78b292a2946d2346a24ab16e49a2f0d69882a2437b09e4f86422590a2ac1b984106b3aea5e3332c96ebc7b6a62b1c35d4b97571b64e96a2d9e909faf160d95bd0446b5119fa0add3b6345e46444dafba3f0e510e1aeb
filename:test06.data
size:512
nhashes:3
hashes:
	d0776891aae896ca05917035a958560027be4b57afc7f6bab7c7088967076d80
	01b63b512a0331aca5ba4408f69fcea26d9a8d979438328822214cefa708171d
	b05a3479f5874246ebed560982eb9b5f2b338ded6951c29138bcc1b9ac12df14
nchunks:4
chunks:
	71d84a15611360c015edd58208e2f38b623ac4a02de4993efb60c86220c8e2ff,0,128
	5f1eb8a0cc1044207565730f360f78fe4bfef42f305f05f1fa83866c1cc50cb8,128,128
	dc32ddaf20b0dc1c6d3eff34c746355ea83c34f498203fbfa8200f5315b642f4,256,128
	afdd69e686866a2c62b37d3f60d4fad4ea13464e6a183d5e7a26c214a8721999,384,128
