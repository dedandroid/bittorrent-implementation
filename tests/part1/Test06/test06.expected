d0776891aae896ca05917035a958560027be4b57afc7f6bab7c7088967076d80
